docs/pages/en-us/overview.md

```diff
@@ -18,20 +18,30 @@ The Audit service provides applications with a reliable, high-performance soluti
 This documented audit trail of user and system activity has many uses.  It provides traceability and individual accountability.  Audit records can also be used for analytical purposes, including security violation detection and abnormal usage patterns.  Audit trails have legal standing and can help protect the organization with a proven record of user and system activity.  Audit records can also be exported for archival and reporting purposes.
 
 ## Introduction
-The Audit Service is designed to provide auditing of user and system actions by defining the required events and the information associated with each event.  An application audit event definition file can be used to generate an application-specific, client-side auditing library.  The Audit service is multi-tenant aware.  Applications send events using the generated client-side auditing library to Elasticsearch where they are indexed according to each tenant.
+
+The Audit Service is designed to provide auditing of user and system actions by defining the required events and the information associated with each event.  An application audit event definition file can be used to generate an application-specific, client-side auditing library.  The Audit service is multi-tenant aware.  Applications send events using the generated client-side auditing library to an endpoint such as the Audit Web Service API or directly to Elasticsearch where they are indexed according to each tenant.
 
 ## User and System Actions
+
 Applications define user and system actions in an audit event definition file.  The definition is utilized to generate a client-side auditing library which raises the defined audit events.
 
 For more details on the audit event definition file, see [Getting Started](Getting-Started).
 
 ## Audit Library
-The audit events definition file is used by a code generation plugin to auto-generate a client-side Java library, which provide type safety and sends audit event messages to Elasticsearch.
+
+The audit events definition file is used by a code generation plugin to auto-generate a client-side Java library, which provides type safety and sends audit event messages to an endpoint.
```

Comment:
- I would put _utilized_ instead of _used_. 
- _enforces_ instead of _provides_.